doc/dev/quick_start.rst

```diff
@@ -636,8 +636,8 @@ decorator to compute Jacobian matrices of general hybrid functions with multiple
         return g(x)
 
 >>> workflow(jnp.array([2.0, 1.0]))
-array([[-1.32116540e-07,  1.33781874e-07],
-       [-4.20735506e-01,  4.20735506e-01]])
+array([[ 1.74393425e-16  4.54648713e-01]
+       [-1.74393425e-16 -4.54648713e-01]])
```

Comment:
do we expect the gradient to change 😅

Reply:
I have copied the wrong one normally there is just a transposition, sorry for that. And this was introduced in a previous PR. Open a Pr shortly.

```
[[ 3.48786850e-16 -4.20735492e-01]
 [-8.71967125e-17  4.20735492e-01]]
```